satpy/readers/abi_l1b.py

```diff
@@ -136,11 +133,11 @@ def _raw_calibrate(self, data):
     def _vis_calibrate(self, data):
         """Calibrate visible channels to reflectance."""
         solar_irradiance = self["esun"]
-        esd = self["earth_sun_distance_anomaly_in_AU"].astype(float)
+        esd = self["earth_sun_distance_anomaly_in_AU"].astype(np.float32)
```

Comment:
Probably not. It was more that I saw the `.astype(float)` and tried changing this first to see if it would make the rest of this magically remain 32-bit instead of jumping to 64-bit. Let's remove it.

```suggestion
        esd = self["earth_sun_distance_anomaly_in_AU"]
```